src/ILCompiler.Compiler/src/Compiler/CompilerTypeSystemContext.BoxedTypes.cs

```diff
@@ -299,8 +295,6 @@ public BoxedValueType(ModuleDesc owningModule, MetadataType valuetype)
                 // mechanism they use for everything else at runtime.
                 // The main difference is that the "Boxed_ValueType" version has no fields. Reference types
                 // cannot have byref-like fields.
-                if (!valuetype.IsByRefLike)
```

Comment:
I left the comment in here.  Not sure about that.

Reply:
I would delete the last part of the comment, starting with "The main difference ...". The rest of the comment is still valid.